plugins/woocommerce/includes/admin/views/html-admin-page-status-report.php

```diff
@@ -762,6 +764,18 @@
 			<td class="help"><?php echo wc_help_tip( esc_html__( 'Is your site enforcing the use of Approved Product Download Directories?', 'woocommerce' ) ); /* phpcs:ignore WordPress.XSS.EscapeOutput.OutputNotEscaped */ ?></td>
 			<td><?php echo wc_get_container()->get( Download_Directories::class )->get_mode() === Download_Directories::MODE_ENABLED ? '<mark class="yes"><span class="dashicons dashicons-yes"></span></mark>' : '<mark class="no">&ndash;</mark>'; ?></td>
 		</tr>
+		<tr>
+			<td data-export-label="Order datastore"><?php esc_html_e( 'Order datastore:', 'woocommerce' ); ?></td>
+			<td class="help"><?php echo wc_help_tip( esc_html__( 'Datastore currently in use for orders.', 'woocommerce' ) ); ?></td>
+			<td><?php echo esc_html( WC_Data_Store::load( 'order' )->get_current_class_name() ); ?></td>
+		</tr>
+		<?php if ( wc_get_container()->get( OrderUtil::class )->custom_orders_table_usage_is_enabled() ) : ?>
```

Comment:
What do you think of (except, we could import `FeaturesController` to make this cleaner/shorter):

```suggestion
		<?php if ( wc_get_container()->get( Automattic\WooCommerce\Internal\Features\FeaturesController::class )->feature_is_enabled( 'custom_order_tables') ) : ?>
```

Otherwise, as I understand it, we are only checking if the custom order tables are authoritative. That could lead to two different scenarios that might be confusing for users (and for HEs):

- If COT is authoritative and sync is enabled, *then later on HPOS is subsequently disabled at feature level,* we will still show this section and will still indicate that sync is enabled (which isn't really true: no sync will take place).
- Or, CPT is authoritative and sync is enabled (and the feature is also enabled), we won't show this section at all—but sync is still taking place.